### PR TITLE
Enable Doctrine cache in prod, avoid extra memcached calls

### PR DESCRIPTION
**Pull request by jocel1**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Enable doctrine cache in prod and avoid unnecessary calls to memcached |
| Type? | improvement |
| Category? | CORE |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? | run tests |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

[*] CORE : Avoid a few memcached set in deleteQuery for blacklisted tables
[*] BO : only calls getForm() when needed

### CODE REVIEW

**Comment by jocel1** [comment]
https://blackfire.io/profiles/compare/51bc8b06-2ffd-4920-92a9-6770fa023438/graph

**Comment by xBorderie** [comment]
Nice stats, thanks!

**Comment by maximebiloe** [comment]
Thanks for this!

Could you amend your commit message to respect the new standards, please?
It should like this: "CO: Enable Doctrine cache in prod"

Thanks

File: app/config/config_prod.yml [hunk 1]
```diff
@@ -25,3 +25,7 @@ monolog:
             level: debug
         console:
             type:  console
+
+doctrine:
+    orm:
+        metadata_cache_driver: apc
```
Comment (mickaelandrieu) [inline app/config/config_prod.yml]: is `apc` already a requirement of PrestaShop ?

**Comment by mickaelandrieu** [comment]
By the way, good job :+1:

File: app/config/config_prod.yml (hunk 1 above)
Comment (jocel1) [inline app/config/config_prod.yml]: I've just pushed a change to use apc only if the extension is available.
Wonder if we should link this behavior with the current cache system (_PS_CACHE_ENABLED_ + _PS_CACHING_SYSTEM_), or leave it as is.

**Pull request merged**